integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -718,12 +719,15 @@ void testUpdateImageName() {
     //print out the original image name
     String imageName = domain1.getSpec().getImage();
     logger.info("Currently the image name used for the domain is: {0}", imageName);
+    logger.info("TEST_WEBLOGIC_IMAGE {0}", TEST_WEBLOGIC_IMAGE);
+    logger.info("DOMAIN_IMAGES_REPO {0}", DOMAIN_IMAGES_REPO);
+    logger.info("WEBLOGIC_IMAGE_NAME {0}", WEBLOGIC_IMAGE_NAME);
 
     //change image name to imageUpdate
     String imageTag = CommonTestUtils.getDateAndTimeStamp();
-    String imageUpdate = KIND_REPO != null ? KIND_REPO
-        + (WEBLOGIC_IMAGE_NAME + ":" + imageTag).substring(TestConstants.BASE_IMAGES_REPO.length() + 1)
-        : DOMAIN_IMAGES_REPO + "/" + WEBLOGIC_IMAGE_NAME + ":" + imageTag;
+    String imageUpdate = KIND_REPO != null 
+         ?  KIND_REPO + ("weblogick8s/test-images/weblogic:" + imageTag)
```

Comment:
why are we hard coding weblogick8s/test-images/weblogic?

Reply:
We push the test images to kind repo to this location. There is no variable name for it in TestConstants class

Reply:
check TestConstants.WEBLOGIC_IMAGE_NAME_DEFAULT